src/main/java/collectionslists/Word.java

```diff
@@ -0,0 +1,24 @@
+package collectionslists;
+
+public class Word {
+    private String word;
```

Comment:
```suggestion
    private final String word;
```
так що ідеально було